files/image_config/monit/container_checker

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
```

Comment:
Can you add a header comment/docstring here describing that this script is intended to be run by Monit, what it does and note that all `print(...)` statements will be appended to the Monit log messages? Something like,

```python3
"""
container_checker 

This script is intended to be run by Monit ...
"""
```

Maybe also add an example of how to run it in a Monit config file?

Reply:
Great suggestion, Updated!